src/app/login/page.tsx

```diff
@@ -1,35 +1,35 @@
 'use client';
 
 import { useState } from 'react';
+import { useRouter } from 'next/navigation';
 import supabase from '../../api/supabase/createClient';
 
 export default function Login() {
   const [email, setEmail] = useState('');
   const [password, setPassword] = useState('');
+  const { push } = useRouter();
   const handleSignUp = async () => {
-    const { data, error } = await supabase.auth.signUp({
+    const { error } = await supabase.auth.signUp({
       email,
       password,
     });
 
     if (error) {
       throw new Error(`An error occurred trying to sign up: ${error}`);
     }
-
-    return data;
+    push("/");
   };
 
   const signInWithEmail = async () => {
-    const { data, error } = await supabase.auth.signInWithPassword({
+    const { error } = await supabase.auth.signInWithPassword({
       email,
       password,
     });
 
     if (error) {
       throw new Error(`An error occurred trying to sign in: ${error}`);
```

Comment:
flagging this here as well for `handleSignIn`

Reply:
got it! thanks for the tip! i've corrected them all to use `error.message`